variables.pkr.hcl

```diff
@@ -87,6 +87,12 @@ variable "source_ami_al2arm" {
   description = "Amazon Linux 2 ARM source AMI to build from."
 }
 
+variable "source_ami_al2_gpu" {
+  type        = string
+  description = "Amazon Linux 2 source AMI to build AL2GPU AMI from. This is a temporary override."
+  default = "amzn2-ami-minimal-hvm-2.0.20230926.0-x86_64-ebs"
```

Comment:
nit: i think static check is failing for this alignment

Reply:
Yep fixed.